doc/article/en-US/the-aurelia-cli.md

```diff
@@ -245,13 +245,18 @@ The Bootstrap example above results in the bundling of the JavaScript portions o
         "exports": "$",
         "resources": [
           "css/bootstrap.css"
-        ]
+        ],
+         "copy": {
+          "fonts/glyphicons-halflings-regular.woff2": "../bootstrap/fonts/glyphicons-halflings-regular.woff2",
+          "fonts/glyphicons-halflings-regular.woff": "../bootstrap/fonts/glyphicons-halflings-regular.woff",
+          "fonts/glyphicons-halflings-regular.ttf": "../bootstrap/fonts/glyphicons-halflings-regular.ttf"
+        }
       }
     ]
   </source-code>
 </code-listing>
 
-Notice that we've added a `resources` array. Here we can provide a list of additional files to be included with the bundle. These files are relative to the `path` designated above and must include the file extension. You can also use glob patterns in place of exact file names.
+Notice that we've added a `resources` array and a `copy` object. In `resources` we can provide a list of additional files to be included with the bundle. These files are relative to the `path` designated above and must include the file extension. You can also use glob patterns in place of exact file names. In `copy` we can specify files to be copied to the output directory. This is very useful when working with libraries that load files via http, which is what Bootstrap does with font files. The `copy` object works as a 'from':'to' setup, where 'from' is the file you want to copy, and 'to' is the destination. Remember to always include the file extension.  
```

Comment:
Could you clarify that "from" is based from the `path` location in the config, and "to" is based on the project root (or wherever you based it)? That will just help with some situations where somebody gets confused about where the paths are coming from.